tests/test_auto_detection.py

```diff
@@ -39,15 +41,17 @@ def test_loop_auto():
     assert type(policy).__module__.startswith(expected_loop)
 
 
-def test_http_auto():
+@pytest.mark.asyncio
```

Comment:
I dont get why those 2 tests suddenly need the async marker

Reply:
the WebsocketProtocol classes call get_event_loop() which is deprecated unless there's a running loop

Reply:
Details here: https://github.com/aaugustin/websockets/issues/988

Reply:
mmmm now I get it thanks